action.yml

```diff
@@ -13,6 +13,8 @@ inputs:
     description: 'Signing key secret retrieved after creating binary cache on https://cachix.org'
   skipPush:
     description: 'Set to true to disable pushing build results to the cache'
+  pushFilter:
+    description: Regular expression to exclude derivations for the cache push, for example "(-source$|nixpkgs\.tar\.gz$)". Warning: this filter doesn't guarantee it won't get pushed in case the path is part of the closure of something that will get pushed.
```

Comment:
Seems like you have to put the description in single quotes

Reply:
Correct, thank you for the hint. Fixed that.